part2/transports/disk/file_trans.py

```diff
@@ -4,16 +4,13 @@
 from thrift.transport import TTransport
 
 class Trade:
-    def __init__(self):
-        symbol=""
-        price=0.0
-        size=0
+    def __init__(self, symbol, price, size):
```

Comment:
Think if we are going to pass them we should use them right? Pylint will squawk here.

Reply:
I fat fingered. I will update it tomorrow morning.

Reply:
Ok, I checked now that indeed it does work:
```
$ python file_trans.py
Trade(86): F 2500 @ 13.100000
```